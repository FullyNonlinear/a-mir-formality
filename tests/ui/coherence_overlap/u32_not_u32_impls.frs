// Test that a positive and negative impl for the same type (`u32`, here) is rejected.
[
    crate core {
        trait Foo {}
        impl Foo for u32 {}
        impl !Foo for u32 {}
    }
]
