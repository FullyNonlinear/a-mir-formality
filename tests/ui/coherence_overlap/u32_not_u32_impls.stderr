Error: check_trait_impl(impl Foo for u32 { })

Caused by:
    failed to disprove
        {! Foo(u32)}
    given
        {}
    got
    {Constraints { env: Env { variables: [], coherence_mode: false }, known_true: true, substitution: {} }}
